.github/workflows/release.yml

```diff
@@ -43,3 +43,19 @@ jobs:
         run: lerna version --yes
         env:
           GH_TOKEN: ${{ secrets.GITHUB_TOKEN }}
+      - name: Generate Lerna Versions PR
+        id: vpr
+        uses: peter-evans/create-pull-request@v4
+        with:
+            token: ${{ secrets.GITHUB_TOKEN }}
+            title: Publish Packages - Lerna
+            body: |
```

Comment:
[curious/question] If we don't specify a `commit-message` arg here, will the PR be generated with the default commit message of "[create-pull-request] automated change" ([docs](https://github.com/marketplace/actions/create-pull-request))? When this PR merges to `master`, we'd ideally be using the commit message specified in the Lerna.json config (i.e., `chore(release): ...`).

If we're generating a PR for the changes with its own distinct commit message now instead of having Lerna generate the commit message, do we possibly need to bring the `chore(release): ...`) commit message into the `commit-message` option for `peter-evans/create-pull-request@v4` instead?

Reply:
Once we get something working I will test this change and see what happens.

Reply:
I've changed this to ${{ github.event.commits[0].message }}